neo/rawio/axonarawio.py

```diff
@@ -336,13 +579,18 @@ def _get_signal_chan_header(self):
         for itetr in range(num_active_tetrode):
 
             for ielec in range(elec_per_tetrode):
-
                 cntr = (itetr * elec_per_tetrode) + ielec
                 ch_name = '{}{}'.format(itetr + 1, letters[ielec])
                 chan_id = str(cntr)
                 gain = gain_list[cntr]
                 stream_id = '0'
-                sig_channels.append((ch_name, chan_id, self.sr, dtype,
+                # the sampling rate information is stored in the set header
+                # and not in the bin file
+                sr = self.file_parameters['set']['sampling_rate']
+                sig_channels.append((ch_name, chan_id, sr, dtype,
                                      units, gain, offset, stream_id))
 
         return np.array(sig_channels, dtype=_signal_channel_dtype)
+
+    def _to_hz(self, param, dtype=int):
```

Comment:
What is the `dtype=int` for ?

Reply:
We should change this to float!

Reply:
I changed the default dtype and used the keyword int at the places where an integer was used earlier